server/routes/survey.router.js

```diff
@@ -1,17 +1,49 @@
+
 const express = require('express');
+const {
+    rejectUnauthenticated,
+} = require('../modules/authentication-middleware');
+const encryptLib = require('../modules/encryption');
 const pool = require('../modules/pool');
+const userStrategy = require('../strategies/user.strategy');
+
 const router = express.Router();
 
-router.get('/', (req, res) => {
-    const queryText = ``;    
-    const queryParams = [];    
+// Handles GET request for ser
+router.get('/', rejectUnauthenticated, (req, res) => {
+    // Send back user object from the session (previously queried from the database)
+    res.send(req.user);
+    console.log('reg.user');
 
 });
 
-router.post('/', (req, res) => {
-    const queryText = ``;    
-    const queryParams = [];    
-});
+// POST endpoint
+router.post('/survey', rejectUnauthenticated, (req, res) => {
```

Comment:
should be just '/' and should route directly to '/api/survey' in server.js

Reply:
One more thing...this should be a PUT endpoint ```router.put``` because its editing an existing value that should be in the able ...see saga.js comment for more